src/Serilog.Sinks.AzureTableStorage/Sinks/AzureTableStorage/AzureBatchingTableStorageSink.cs

```diff
@@ -83,6 +89,7 @@ public class AzureBatchingTableStorageSink : PeriodicBatchingSink
                 throw new ArgumentException("batchSizeLimit must be between 1 and 100 for Azure Table Storage");
 
             _formatProvider = formatProvider;
+            _textFormatter = textFormatter == null ? new CompactJsonFormatter() : textFormatter;
```

Comment:
I don't think we can switch to `CompactJsonFormatter` by default here, since it'd break a lot of existing setups that rely on the `JsonFormatter` structure.